src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type ListOfMovies = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<ListOfMovies> = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map(movie => {
+        return (
+          <div className="card" key={movie.imdbId}>
+            <MovieCard {...movie} />
+          </div>
```

Comment:
add this class card inside of MovieCard

```suggestion
          <MovieCard {...movie} key={movie.imdbId}/>
```